integration/init_app/docker-asset/expected/.ship/release.yml

```diff
@@ -0,0 +1,16 @@
+---
+assets:
+  v1:
+    - docker:
+        image: busybox:latest
```

Comment:
`replicated/helloscratch:1.0.0` is used elsewhere (in the `base` tests) and is only 12k instead of 1.3M
We could create a private version to use as well